CHANGELOG.md

```diff
@@ -1,5 +1,198 @@
 # Changelog
 
+## v1.0.0 (2018-2-7)
```

Comment:
We should add a couple words about
https://github.com/DevExpress/testcafe/pull/3356
https://github.com/DevExpress/testcafe/pull/3177
Also I'm not sure we should put the breaking changes before new features.